src/logic/migrationSql/index.ts

```diff
@@ -0,0 +1,31 @@
+import { PrismaTemplates } from '../../'
+import migrations from '../../generatedMigrations/index'
+import { BaseTemplateParametersResolved } from '../../types'
+
+export type MigrationSqlResponse = string[]
+
+export type Params = {
+  template: PrismaTemplates.$Types.TemplateTag
+  parameters: BaseTemplateParametersResolved
+}
+
+export const select = (params: Params): MigrationSqlResponse => {
+  if (
+    params.parameters.datasourceProvider === 'sqlite' ||
+    params.parameters.datasourceProvider === 'mongodb' ||
+    params.template === 'Empty'
+  ) {
+    return []
+  }
+  if (params.parameters.datasourceProvider === 'mysql') {
+    // Planetscale mode
+    if (params.parameters.referentialIntegrity === 'prisma') {
+      return migrations[`${params.template}${params.parameters.datasourceProvider}`]
```

Comment:
Is this lookup type safe? The `?` below suggests to me it is not as much as it could be.

Reply:
It is typesafe, the `?` shouldn't be there. I removed it but this will soon probably change anyways